### PR TITLE
fix cron_path this way?

### PR DESCRIPTION
**Pull request by damm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ Actions:
     <td><tt>cron_path</tt></td>
     <td>String</td>
     <td>sets the PATH variable in the crontab to specify who should get the output of the crontab run</td>
-    <td></td>
+    <td>/usr/bin:/bin/usr/local/bin:/opt/chef/embedded/bin</td>
```
Comment (patcon) [inline README.md]: missing colon after "/bin" :)

Comment (patcon) [inline README.md]: Possssssibly because I was as well in #35 :/

**Comment by patcon** [comment]
+1 otherwise. thanks for helping to get a fix in place quickly damm

**Comment by damm** [comment]
Thank you for filing the issue and helping me fix it :)  Have a good weekend.

**Comment by ramonskie** [comment]
this produces "PATH=/usr/bin:/bin:/usr/local/bin:/opt/chef/embedded/bin:/opt/chef/embedded/bin"

ruby path is set 2 times

**Pull request merged**